README.md

```diff
@@ -53,17 +57,17 @@ An example of this can be found in the [Decentraland repository](https://github.
         "codegen": "graph generate-types subgraph.yaml",
         "build": "graph build subgraph.yaml",
         "build-ipfs": "graph build --ipfs /ip4/127.0.0.1/tcp/5001 subgraph.yaml",
-        "deploy": "graph deploy subgraph.yaml --watch --verbosity debug --node http://35.242.224.197:8020/ --ipfs /ip4/35.242.224.197/tcp/8030 --subgraph-name my-subgraph"
+        "deploy": "graph deploy subgraph.yaml --watch --verbosity debug --node http://<YOUR_IP_ADDRESS> --ipfs /ip4/<YOUR_IP_ADDRESS>/tcp/8030 --subgraph-name my-subgraph"
```

Comment:
Small fix: I would put IP_ADDRESS:PORT and explain below what that means:
- IP_ADDRESS is the IP address of your own node (for example, 35.242.224.197)
- PORT is the port number on which the node is running (for example. 8020)

Or something along those lines. So it's more clear :)